src/probe/probe.c

```diff
@@ -14,13 +14,18 @@
 #include <sof/lib/alloc.h>
 #include <sof/lib/dma.h>
```

Comment:
@abonislawski please add the message you added in the PR description to this commit description. Commit description is kept in git history while PR will be lost in the infinity of github.

Reply:
Also add the important "Fixes: 8b7275b65f17d6" reference in the commit message.

Reply:
@dbaluta @marc-hb done

Reply:
It's commit 0529c141f40b, my bad.  8b7275b is probably an earlier, not merged version (of the same commit)
Please use 12 characters `0529c141f40b` as recommended by `./scripts/checkpatch.pl --no-tree`

`checkpatch` isn't always right but these other `checkpatch` warnings seem easy to fix too.

Reply:
Ok fine, updated

Reply:
@abonislawski correct way of referencing commit is also specifying the commit subject.

e.g Fixes: COMMIT_HASH ("Commit subject")

Reply:
@abonislawski also look at the checkpatch.pl warnings/errors. Are they addressable?